core/domain/learner_playlist_services_test.py

```diff
@@ -100,62 +101,76 @@ def _get_all_learner_playlist_collection_ids(self, user_id):
             learner_playlist_model.collection_ids if
             learner_playlist_model else [])
 
-    def test_mark_exploration_to_be_played_later(self):
+    def test_already_subscribed_to_exploration(self):
```

Comment:
When naming tests, follow the guidelines in step 2 here: https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests

It should be possible to tell what the test is doing from the test name. In this case I don't know what action is being tested. I think you want something like test_that_adding_subscribed_exps_to_learner_playlist_does_nothing() or similar.

Ditto below, for all test names. Be as clear as possible.

Reply:
Got it. Thanks!